src/perform_step/firk_perform_step.jl

```diff
@@ -305,181 +323,38 @@ end
     integrator.force_stepfail = true
     return
   end
-  cache.ηold = η
-  cache.iter = iter
 
   @. u = uprev + z2
-  f(fsallast, u, p, t+dt)
-  return
-end
-
-@muladd function perform_step!(integrator, cache::RadauIIA5ConstantCache, repeat_step=false)
```

Comment:
why is this deleted?

Reply:
deleted it by accident, just fixed in the the next commits